content-src/asrouter/templates/NewsletterSnippet/NewsletterSnippet.schema.json

```diff
@@ -43,6 +43,11 @@
       "type": "string",
       "description": "Endpoint to submit form data."
     },
+    "form_method": {
```

Comment:
Is it really necessary to make this user configurable? Instead, why not just do something like this:

```js
// Where NewsletterSnippet is renamed to the generic SubmitFormSnippet:
const NewsletterSnippet = props => <SubmitFormSnippet {...props} form_method="POST" />;
const FXASnippet = props => <SubmitFormSnippet {...props} form_method="GET" />;
...
```